nh_eobs_mental_health/static/src/js/nhlib.js

```diff
@@ -1202,17 +1202,21 @@ NHMobilePatient = (function(superClass) {
   };
 
   NHMobilePatient.prototype.drawGraph = function(self, serverData, dataModel) {
-    var chartEl, chartFunc, chartFuncName, controls, graphContent, graphTabs, tableEl, tableFunc, tableFuncName, validChart, validTable;
+    var activeTab, chartEl, chartFunc, chartFuncName, controls, graphContent, graphTabs, tableEl, tableFunc, tableFuncName, validChart, validTable;
     graphContent = document.getElementById("graph-content");
     controls = document.getElementById("controls");
     chartEl = document.getElementById(self.chart_element);
     tableEl = document.getElementById(self.table_element);
     graphTabs = graphContent.parentNode.getElementsByClassName("tabs")[0];
+    activeTab = graphTabs.getElementsByClassName("selected")[0].getAttribute('href');
     chartFuncName = "draw" + dataModel.capitalize() + "Chart";
     tableFuncName = "draw" + dataModel.capitalize() + "Table";
     if (serverData.length > 0) {
       controls.style.display = "block";
       graphTabs.style.display = "block";
+      chartEl.style.display = "block";
+      graphContent.style.display = "block";
+      tableEl.style.display = "block";
```

Comment:
Could these (and any other JS changes to style) be bundled up into a CSS class? That way the JS only needs to apply a class, separates concerns a bit more.

Also if there's any other bits like this might be worth maintaining a list so you can do:
```javascript
for (var i = 0; i < graphElements.length; i++) {
    element = graphElement[i]
    element.style.display = "block";
}